test/spec/differ.js

```diff
@@ -357,20 +380,20 @@ describe('diffing', function() {
         // then
         expect(results._added).to.have.keys([
           'ManualTask_1',
-          'ExclusiveGateway_1'
+          'ExclusiveGateway_1',
+          'undefined'
```

Comment:
I believe this is the correct behaviour as the time definition does not have any `Date` value added. Please let me know if this is incorrect :)